package.json

```diff
@@ -46,6 +46,35 @@
           "import/first": "off",
           "import/no-duplicates": "off"
         }
+      },
+      {
+        "files": [
+          "*.js"
+        ],
+        "rules": {
+          "import/no-cycle": "off",
```

Comment:
Should this warn? Ideally these would be resolved

Reply:
The linter rules will be reviewed after the migration:
https://github.com/cerner/terra-graphs/issues/8